docs/devguide/create-metricset.asciidoc

```diff
@@ -1,319 +0,0 @@
-[[creating-metricsets]]
-=== Creating a Metricset
```

Comment:
This is developer documentation for Metricbeat. I think we should keep this, new metricsets may still be needed to act as new metric inputs.